### ISSUE TITLE
Option to Search for Item in Group Filter

### ISSUE THREAD

**Issue by FourTOne5** [tachiyomiorg/tachiyomi#5539]
### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have updated the app to version **[0.11.1](https://github.com/tachiyomiorg/tachiyomi/releases/tag/v0.11.1)**.
- [X] I will fill out all of the requested information in this form.

### Describe your suggested feature

When Group Filter will be expanded there will search at top. If the search query stays same for 2 - 3 seconds (Implementing a search button will be tedious.). The group items will be filtered by that query.

**Comment by CarlosEsco** [comment]
Wouldn't the UI for this be confusing?  There are already search fields for searching the actual source in the filter sheet.  Now there would be a search field that only searches in the filter

**Comment by FourTOne5** [comment]
I don't think it will be much confusing. The proposed UI will be something like this.

While Expanded:
![Screenshot_20210712_195204](https://user-images.githubusercontent.com/59261191/125299278-d2e15580-e34a-11eb-8705-a273181ff2e7.jpg)

While Not Expanded:
![Screenshot_20210712_195234](https://user-images.githubusercontent.com/59261191/125299457-fd331300-e34a-11eb-9239-45c4d6711e09.jpg)
